src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/UpgradeAssistantHostExtensions.cs

```diff
@@ -102,14 +102,6 @@ public static IHostBuilder UseUpgradeAssistant<TApp>(this IHostBuilder host, IUp
                         }
                     });
 
-                    services.AddNuGet(optionss =>
-                    {
-                        if (upgradeOptions.Project?.FullName is string fullname)
-                        {
-                            optionss.PackageSourcePath = Path.GetDirectoryName(fullname);
```

Comment:
I believe the setting of `PackageSourcePath` got lost in this PR. It would explain why changing package sources in `nuget.config` doesn't work anymore.

Reply:
I got the same issue, 0.4.421302 will not follow my nuget.config but 0.2.241603 (a version that is before the above PR) will follow my nuget.config.